### ISSUE TITLE
Google Drive

### ISSUE THREAD

**Issue by hampoelz** [alex-shpak/keemob#6]
Hey, could you please add Google Drive Support?

**Comment by alex-shpak** [comment]
Hi!

I'm using app with google drive via content providers.
When you select file navigate to side menu (drawer) and select there Google Drive

Right now getting file via google drive is disabled ([here](https://github.com/keeweb/keeweb/blob/master/app/scripts/storage/index.js#L16)) due to bad UX and additional battery impact. Basically you will have to login into google again with GDrive storage enabled.
Enabling it, on another hand, will improve merge functionality.

If you think it's important to have - I can look into it.

Also https://github.com/alex-shpak/keemob/issues/3

**Comment by hampoelz** [comment]
Hi,

I would really need this function

I have already tried to open the file with 'side menu/Google Drive' but when I make changes (for example add a new password) the file does not save / the sync does not work

**Comment by alex-shpak** [comment]
I see,
Can you please write your steps?

Such use case works for me via `ContentProvider`
1. Add item in mobile app
2. Press `logout` icon
3. Login at desktop in around 10secs
4. I can see just created item there.

In reverse direction desktop -> mobile it takes more time to sync.

**Comment by hampoelz** [comment]
The sync from PC to Phone works... , but when I open the file on the phone from Google drive and I make and save the changes, and reopen the file, the changes will be lost

**Comment by iamrealvinod** [comment]
Could you please fix the sync issue? Any changes that we make in mobile are not getting synced back to google drive. Also i am not able to find the file in mobile to sync it manually to google drive. Please help @alex-shpak !

**Comment by KaKi87** [comment]
Hello,
Any news on this ?
Thanks !

**Comment by marie-bnl** [comment]
Omg hello KaKi I didn't think I'd find you there 😂😂

Anyway, I think it would be very cool to have network drives as I use Dropbox only for my kdbx so I'm not going to install the Dropbox app only for this

**Comment by iamrealvinod** [comment]
@alex-shpak any update on this please?

**Comment by alex-shpak** [comment]
Hi @iamrealvinod!
Unfortunately cordova launcher was removed from keeweb, so I can't build new version now :( 

https://github.com/keeweb/keeweb/commit/eab683ec718148a62d4b41cd9d262de48c60fa87#diff-58db5bcce493b8d9f9c82a304f0c0f0396415a4437b146967ead69e9a0b21ea0

Unless I find out how to make it via keeweb extension I can't build a newer verson now.

**Comment by hampoelz** [comment]
If I find the time, I could create an Android PWA wrapper for Keeweb so you can use Google Drive and all the other features of the Keeweb web application as app.

**Comment by alex-shpak** [comment]
Well, for that you can install KeeWeb as PWA from browser.

**Comment by hampoelz** [comment]
Yeah, I know, but it doesn't have features like fingerprint support like in Keemob.

**Comment by alex-shpak** [comment]
Yes, unfortunately.
But PWA would not solve that.

**Comment by hampoelz** [comment]
A PWA wrapper could invoke some js in the web view to allow the site to communicate with native code.
